user/validation/explicit/plate-hole-iso/plate-hole-iso.md

```diff
@@ -0,0 +1,140 @@
+# Cylindrical Hole in an Infinite Mohr-Coulomb Medium
+
+A square plate is modeled twice in the plane strain condition using quarter-symmetry. The first plate has length and width of 10 m and the second plate has length and width of 30 m. A quarter circle of radius 1 m is removed from the bottom left corner of both plates. Boundary conditions including fixed boundaries and external loads are described in Figure 1. External pressure is $P_0$ = -30 MPa. Additionally, the plate is subjected to an insitu stress of $P_0$ = -30 MPa (compression negative).
+
+![Boundary Conditions](plate-hole-iso.svg)
+> Figure 1: Load and boundary conditions for 10 m and 30 m models
+
+
+## Analytical Solution
+
+The closed form solution for this problem is from Salencon [1]. The normalized form of the analytical solution is from [2]. 
+
+### Elastic-plastic boundary
+
+The yield zone radius ($R_0$) is
+
+$$\frac{R_0}{a} = \left(\frac{2}{K_p+1}\frac{1+\frac{q}{P_0}k_p}{\frac{P_i}{P_0}+\frac{q}{P_0}k_p}\right)^{k_p}$$
```

Comment:
@jgiven100 Will mark down convert the equation written using latex format?

Reply:
@bodhinandach Yes equations will look good using Latex in an `.md` doc. Although I'm not sure if that for all github pages or a particular setting on our cb-geo site

Reply:
https://deploy-preview-50--mpm.netlify.app/#/user/validation/explicit/plate-hole-iso/plate-hole-iso